.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Change description
+
+> Description here
+
+## Type of change
+
+- [ ] Bug fix (fixes an issue)
+- [ ] New feature (adds functionality)
+
+## Related issues
+
+Does this PR fix a Github Issue?
+
+> Fix [#1]()
+
+## Checklists
+
+### Development
+
+- [ ] Application changes have been tested thoroughly
+
+### Security
```

Comment:
Probably called `Impact`, so below we can ask about security, performance, migrations, etc.